net.ptilouk.littlebrats.yml

```diff
@@ -0,0 +1,29 @@
+app-id: net.ptilouk.littlebrats
+base: org.godotengine.godot.BaseApp
+base-version: '4.4'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: little-brats
+rename-icon: little-brats
+rename-desktop-file: little-brats.desktop
+rename-appdata-file: little-brats.metainfo.xml
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
```

Comment:
what is network for?

Reply:
This is for online multiplayer (unless I misunderstood what network access was, I think it is required in that case?)